cmd/endpoint.go

```diff
@@ -142,6 +148,15 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 	}, nil
 }
 
+// isHostIPv4 - helper for validating if the provided arg is an ip address.
+func isHostIPv4(ipAddress string) bool {
```

Comment:
As this function is introduced in this PR, I couldn't able to understand why we need another PR for moving it to `net.go`